debugtools/DDR_VM/src/com/ibm/j9ddr/tools/StructureStubGenerator.java

```diff
@@ -196,7 +197,14 @@ private static void writeClassComment(PrintWriter writer, String name) {
 		writer.println(" */");
 	}
 
-	private static void writeOffsetStubs(PrintWriter writer, StructureDescriptor structure) {
+	private String getOffsetConstant(FieldDescriptor fieldDescriptor) {
+		if (opts.get("-l").equals("true")) {
+			return PointerGenerator.getOffsetConstant(fieldDescriptor.getName());
```

Comment:
Please extract `fieldDescriptor.getName()` to a local variable at the call site and change this method so it receives the field name.